feat: Add a new cluster state "Unhealthy" (#63)
stackabletech/trino-lb: CHANGELOG.md

- Support configuring the scaler reconcile interval ([#61]).
- Add simple web-based dashboard that shows the current state and query counts of all clusters.
  This makes it easier to debug state transitions of clusters ([#62]).

> - Add a new cluster state `Unhealthy`.
>   This state is entered once the readiness check of a `Ready` cluster fails (the check is implemented in the scaler implementation).
>   The `Unhealthy` state is kept until the scaler marks that Cluster as ready again.
>   `Unhealthy` clusters won't get any new queries, if all clusters are unhealthy, queries are queued.\
>   Note: Use the now configurable scaler reconcile interval to detect cluster changes quickly ([#63]).

Reviewer: suggestion: Slight rewording.

```suggestion
- Add `Unhealthy` cluster state.
  This state is entered once the readiness check of a cluster in the `Ready` state fails.
  The cluster will remain in the `Unhealthy` state until the scaler marks that cluster as `Ready` again.
  `Unhealthy` clusters won't get any new queries; if all clusters are unhealthy, queries will remain in the queue.
  The cluster health check interval can be configured using the scaler reconcile interval ([#63]).
```

Author: Please have a look at c2959b1

>   <br>Note: Use the now configurable scaler reconcile interval to detect cluster changes quickly ([#63]).

Reviewer: note: I don't quite understand the sentence in line 15. It is either wrong, or worded weirdly.

note: markdownlint complains about the `<br />` tag. I think we should remove it.

Author: You are right, there was a typo: a9726af

Author: TIL a backslash also works: 5e5b022
